### PR TITLE
Update document about UART

### PR DESCRIPTION
**Pull request by sae-bom**
* Make document explains UART API design
* Update document for RPi with how to enable UART module

IoT.js-DCO-1.0-Signed-off-by: Saebom Kim sae-bom.kim@samsung.com

### CODE REVIEW

File: docs/IoT.js-API-UART.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+```
+
+
+### Methods
+
+#### uart.open(callback)
+* `callback: Function(err)`
+
+Open UART device.
```
Comment (glistening) [inline docs/IoT.js-API-UART.md]: Could you change Open to Opens? Currently our API documents use third person singular verbs in the simple present tense.

File: docs/IoT.js-API-UART.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+
+#### uart.write(data, callback)
+* `data: String`
+* `callback: Function(err)`
+
+Write data to UART device.
```
Comment (glistening) [inline docs/IoT.js-API-UART.md]: Ditto. Please change Write to Writes.

File: docs/IoT.js-API-UART.md (hunk 1 above)
Comment (sae-bom) [inline docs/IoT.js-API-UART.md]: OK, I see

**Comment by yichoi** [comment]
LGTM

**Pull request merged**